### PR TITLE
Tappx: Change full dinamic endpoint to a parcial dinamic endpoint

### PR DESCRIPTION
**Pull request by TappxJon**

### CODE REVIEW

File: adapters/tappx/tappx.go [hunk 1]
```diff
@@ -131,12 +131,12 @@ func (a *TappxAdapter) buildEndpointURL(params *openrtb_ext.ExtImpTappx, test in
...
-	isNewEndpoint, err := regexp.Match(`^(zz|vz)[0-9]{3,}([a-z]{2}|test)$`, []byte(params.Endpoint))
+
+	isNewEndpoint, err := regexp.Match(`^(zz|vz)[0-9]{3,}([a-z]{2,3}|test)$`, []byte(params.Endpoint))
 	if isNewEndpoint {
-		tappxHost = params.Endpoint + ".pub." + tappxHost + "/rtb/"
+		tappxHost = params.Endpoint + ".pub.tappx.com/rtb/"
 	} else {
-		tappxHost = "ssp.api." + tappxHost + "/rtb/v2/"
+		tappxHost = "ssp.api.tappx.com/rtb/v2/"
 	}
```
Comment (SyntaxNode) [inline adapters/tappx/tappx.go:140]: There are two kinds of endpoints for this adapter. Can you standardize on one? In that choice, the "old endpoint" is preferred by Prebid.

Please see this thread: https://github.com/prebid/prebid-server/issues/2606

**Review by SyntaxNode — COMMENTED** [review]
Thank you for responding and acting quickly to resolve the host macro usage. We'll ask a few questions first to understand your situation and offer the best solution.

File: adapters/tappx/tappx.go (hunk 1 above)
Comment (prebidtappx) [reply adapters/tappx/tappx.go:140]: Hello @SyntaxNode,
In short term we still need both, but we are planning at mid term to leave only one.
But the one we plan to leave is the "new endpoint", I know this is more "dynamic" but we need to adapt the endpoint to each Publisher in order to work correctly.

**Review by Sonali-More-Xandr — DISMISSED** [review]
LGTM!

**Comment by Sonali-More-Xandr** [comment]
Please resolve the merge conflicts.

**Review by onkarvhanumante — COMMENTED** [review]
> @TappxJon could you respond back on [#2624 (comment)](https://github.com/prebid/prebid-server/pull/2624#discussion_r1126631519)
>> Hello @SyntaxNode, In short term we still need both, but we are planning at mid term to leave only one. But the one we plan to leave is the "new endpoint", I know this is more "dynamic" but we need to adapt the endpoint to each Publisher in order to work correctly.

thank you @prebidtappx

**Review by onkarvhanumante — CHANGES_REQUESTED** [review]
Changes looks good to me. @TappxJon  could you help with resolving conflict on `adapters/tappx/tappx.go`

**Comment by onkarvhanumante** [comment]
> Changes looks good to me. @TappxJon could you help with resolving conflict on `adapters/tappx/tappx.go`

@TappxJon  @prebidtappx

**Comment by prebidtappx** [comment]
Hello @onkarvhanumante,
Sorry for taking so long. Conflicts resolved and all checks passed. Thank you!

**Review by Sonali-More-Xandr — APPROVED** [review]

**Review by onkarvhanumante — APPROVED** [review]

**Pull request merged**